cmd/protoc-gen-gorums/gengorums/gorums.go

```diff
@@ -203,6 +277,27 @@ func hasMethodOption(method *protogen.Method, methodOptions ...*protoimpl.Extens
 	return false
 }
 
+// hasAllMethodOption returns true if the method has all of the given method options
+func hasAllMethodOption(method *protogen.Method, methodOptions ...*protoimpl.ExtensionInfo) bool {
+	ext := protoimpl.X.MessageOf(method.Desc.Options()).Interface()
+	for _, callType := range methodOptions {
+		if !proto.HasExtension(ext, callType) {
+			return false
+		}
+	}
+	return true
+}
+
+// hasStrictOrderingOption returns true if the method has one of the given strict ordering method options.
+func hasStrictOrderingOption(method *protogen.Method, methodOptions ...*protoimpl.ExtensionInfo) bool {
+	for _, option := range methodOptions {
+		if strictOrderingTypeCheckers[option](method) {
```

Comment:
This will fail if one of the provided `methodOptions` is not in the map. Replace with:
```go
	if f, ok := strictOrderingTypeCheckers[option]; ok && f(method) {
```